lib/order.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'menu'
+require_relative 'customer'
+include Menu
+include Customer
+
+class Order
+
+attr_reader :basket, :select_items, :change_quantity, :basket_count, :order_value, :total_values
+
+  def initialize
+    @order = {}
+    @min_order_value = 10
+  end
+
+  def select_items(item)
+    Menu.items {|k,v| k.match item}
+      @order.merge!(item)
+  end
+
+  def change_quantity(item)
+  end
+
+  def basket
+    @order
+  end
+
```

Comment:
If your instance variable is called order, and you have an attr_reader :order, then you do not need another method to read the order. If you wanted to manually write the attr_reader, the convention would be for the method to have the same name as the variable.

``` attr_reader :order ```

is the same as:

```
def order
  @order 
end
```